pkgs/applications/misc/etesync-dav/default.nix

```diff
@@ -2,31 +2,27 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "etesync-dav";
-  version = "0.20.0";
+  version = "0.30.6";
 
   src = python3Packages.fetchPypi {
     inherit pname version;
-    sha256 = "1q8h89hqi4kxphn1g5nbcia0haz5k57is9rycwaabm55mj9s9fah";
+    sha256 = "0cjz4p3a750fwvrxbzwda0sidw7nscahvppdshbsx49i6qrczpbg";
   };
 
   postPatch = ''
-    substituteInPlace setup.py --replace "Radicale==" "Radicale>="
+    substituteInPlace setup.py
   '';
 
   propagatedBuildInputs = with python3Packages; [
+    etebase
     etesync
     flask
     flask_wtf
     radicale3
   ];
 
-  checkInputs = with python3Packages; [
-    pytest
-  ];
-
-  checkPhase = ''
-    pytest
-  '';
+  doCheck = false;
```

Comment:
Either activate doCheck or do an pythonImportsCheck.

Reply:
0.30.6 doesn't come with any tests and `pythonImportsCheck = [ "etesync-dav" ]` fails. What do you recommend the best solution is in this case?

Reply:
nixpkgs also doesn't yet package etebase/etesync, so I am unsure how you would write a test.

Reply:
I wrote a nix test myself, but I still left `doCheck = false`. Is this fine?

Reply:
> 0.30.6 doesn't come with any tests

Maybe they do not ship them on pypi. You could try the official tarball release.

> and `pythonImportsCheck = [ "etesync-dav" ]` fails.

Oh that can happen with applications which are not supposed to be imported. I thought it was a module.

> What do you recommend the best solution is in this case?

Just disable the tests or try the source repo tarball.

> I wrote a nix test myself, but I still left doCheck = false. Is this fine?

Yeah, this is fine.